packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -198,47 +236,15 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
 
   /**
    * @notice A constructor for initialising a new instance of a Releasing Schedule contract.
-   * @param releaseStartTime The time (in Unix time) at which point releasing starts.
-   * @param releaseCliffTime Duration (in seconds) after `releaseStartTime` of the golds' cliff.
-   * @param numReleasePeriods Number of releasing periods.
-   * @param releasePeriod Duration (in seconds) of each release period.
-   * @param amountReleasedPerPeriod The released gold amount per period.
-   * @param revocable Whether the release schedule is revocable or not.
-   * @param _beneficiary Address of the beneficiary to whom released tokens are transferred.
-   * @param _releaseOwner Address capable of revoking, setting the liquidity provision
-   *                      and setting the withdrawal amount.
-   *                      0x0 if grant is not subject to these operations.
-   * @param _refundAddress Address that receives refunded funds if contract is revoked.
-   *                       0x0 if contract is not revocable.
-   * @param subjectToLiquidityProvision If this schedule is subject to a liquidity provision.
-   * @param initialDistributionRatio Amount in range [0, 1000] (3 significant figures)
-   *                                 indicating % of total balance available for distribution.
-   * @param _canValidate If this schedule's gold can be used for validating.
-   * @param _canVote If this schedule's gold can be used for voting.
-   * @param registryAddress Address of the deployed contracts registry.
+   * @param config Release gold configuration parameters.
    */
-  function initialize(
-    uint256 releaseStartTime,
-    uint256 releaseCliffTime,
-    uint256 numReleasePeriods,
-    uint256 releasePeriod,
-    uint256 amountReleasedPerPeriod,
-    bool revocable,
-    address payable _beneficiary,
-    address _releaseOwner,
-    address payable _refundAddress,
-    bool subjectToLiquidityProvision,
-    uint256 initialDistributionRatio,
-    bool _canValidate,
-    bool _canVote,
-    address registryAddress
-  ) external initializer {
+  function initialize(ReleaseGoldConfig calldata config) external initializer {
```

Comment:
This change is needed since I was not able to come up with the solution that would not give an error:

Stack too deep when compiling inline assembly: Variable value0 is 2 slot(s) too deep inside the stack.

While initializing ReleaseGold smart contract. If you would know about some workaround, please feel free to share it. 

Because of this it seems that ReleaseGold smart contract would be great candidate for migration to 0.8 since 0.5 doesnt officially support structs as function input parameters